inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -398,6 +398,11 @@ void GNAGraphCompiler::ConvolutionPrimitive(InferenceEngine::CNNLayerPtr layer)
         ptr_weights,
         ptr_biases);
 
+    if (inputs->getLayout() == Layout::NHWC) {
```

Comment:
it is possible to move this inside initConvolutionPrimitive - providing layout as new arg

Reply:
@admitriev-gna, please create a new PR to address this comment.